shared/actions/team-building.tsx

```diff
@@ -191,6 +191,18 @@ const fetchUserRecs = async (
   }
 }
 
+async function searchEmailAddress(state: TypedState, {payload: {namespace}}: SearchOrRecAction) {
```

Comment:
Searching an email address is a sort of special case of searching a service, so I thought it made sense to create a separate action out of it so we don't have to deal with extra results or searching the query against all services.